src/lib389/lib389/monitor.py

```diff
@@ -1,12 +1,13 @@
 # --- BEGIN COPYRIGHT BLOCK ---
-# Copyright (C) 2021 Red Hat, Inc.
+# Copyright (C) 2025 Red Hat, Inc.
 # All rights reserved.
 #
 # License: GPL (version 3 or any later version).
 # See LICENSE for details.
 # --- END COPYRIGHT BLOCK ---
 
 import copy
+import psutil
```

Comment:
`mock` build fails because of the missing dependency. So we might need to introduce it...
Probably both, - `setup.py` addition and `pythoc3-psutil` package.